fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImplSpec.groovy

```diff
@@ -14,38 +14,52 @@ import spock.lang.Specification
 import java.util.function.Supplier
 
 class AsyncDruidWebServiceImplSpec extends Specification {
-    def "Ensure that headersToAppend are added"() {
+    def "Ensure that headersToAppend are added to request when calling postDruidQuery"() {
         setup:
         WeightEvaluationQuery weightEvaluationQuery = Mock(WeightEvaluationQuery)
         QueryContext queryContext = Mock(QueryContext)
-        weightEvaluationQuery.getContext() >> { queryContext }
-        queryContext.numberOfQueries() >> { 1 }
-        queryContext.getSequenceNumber() >> { 1 }
```

Comment:
Because of [default behavior of mocks](http://spockframework.org/spock/docs/1.0/interaction_based_testing.html#_default_behavior_of_mock_objects), so it will just return the default type of whatever it returns.